data/everydaycodings.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "Kumar Saksham(Everydaycodings)",
+  "displayStatsPublic": false,
+  "type": "personal",
+  "bio": "Freelance Python Developer | **Aspiring Machine Learning Engineer**| Medium Writer",
+  "avatar": "https://github.com/everydaycodings.png",
+  "tags": ["Python Developer", "Machine Learning Engineer", "Tech Writer", "Open Source"],
+  "socials": [
+    { "icon": "FaLinkdin", "url": "https://www.linkedin.com/in/kumar-saksham1891" },
+    { "icon": "FaGithub", "url": "https://github.com/everydaycodings" }
+  ],
+  "links": [
+    {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/kumar-saksham1891",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "GitHub",
+      "url": "https://github.com/everydaycodings",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Medium",
+      "url": "https://everydaycodings.medium.com",
+      "icon": "FaMedium"
+    },
+    {
+      "name": "Kaggle",
+      "url": "https://www.kaggle.com/everydaycodings",
+      "icon": "FaKaggle"
+    },
+    {
+      "name": "Twitter",
+      "url": "https://twitter.com/everydaycodings",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Instagram",
+      "url": "https://twitter.com/everydaycodings",
```

Comment:
this should be your Instagram link instead of Twitter link

Reply:
Oh i am sorry my bad.